### ISSUE TITLE
Make dashboards embeddable

### ISSUE THREAD

**Issue by Srumith1** [Graylog2/graylog2-server#4313]
Externalize/embed Graylog dashboards

## Expected Behavior

Graylog dashboard should provide some kind of HTML script which can be embedded in any external web-application to display the same dashboard.

Similar feature in Kibana: https://www.elastic.co/guide/en/kibana/5.6/sharing-dashboards.html

## Current Behavior

Graylog does not provide any HTML source to embed dashboards in external applications.

**Comment by nzkozar** [comment]
A much needed feature indeed.

**Comment by simobonazz** [comment]
Any update on this ? It would be really nice to have this feature.
Without this feature it's impossible to integrate graylog dashboard inside existing business application.

**Comment by egpotus** [comment]
A much needed feature

**Comment by fknorn** [comment]
Exactly! Essentially every NOC has a bunch of screens showing different kind of monitoring dashboards. How can I get Graylog's up there too??

**Comment by kmerz** [comment]
Would first need #4824

**Comment by Emy-01** [comment]
After Product review, this ticket will now be closed as it is not currently planned.
